src/System.Drawing.Common/src/System/Drawing/Graphics.Windows.cs

```diff
@@ -2217,103 +1865,77 @@ public void DrawImageUnscaledAndClipped(Image image, Rectangle rect)
             DrawImage(image, rect, 0, 0, width, height, GraphicsUnit.Pixel);
         }
 
-        /*
-         * Affine or perspective blt
-         *  destPoints.Length = 3: rect => parallelogram
-         *      destPoints[0] <=> top-left corner of the source rectangle
-         *      destPoints[1] <=> top-right corner
-         *       destPoints[2] <=> bottom-left corner
-         *  destPoints.Length = 4: rect => quad
-         *      destPoints[3] <=> bottom-right corner
-         *
-         *  @notes Perspective blt only works for bitmap images.
-         */
+        // Affine or perspective blt
```

Comment:
Consider using `///` -style comments and appropriate tags

Reply:
This one was a little weird. It was a general comment that applied to a number of methods.